docs/models/audio.md

```diff
@@ -0,0 +1,26 @@
+# Audio
+
+## Media and File Types
+
+Islandora Defaults uses the built-in Drupal "Audio" media type for video. Islandora Defaults configures it to accept files of type `mp3`, `wav`, and `aac`.
+
+## Derivatives
+
+Islandora uses Homarus (ffmpeg as a microservice) to create audio derivatives. Islandora Defaults sets you up to create:
+
+* "Service file" from the original file, with parameters `-codec:a libmp3lame -q:a 5`, stored in the public filesystem
```

Comment:
(these parameters can be changed in the configuration for the Drupal Action that Islandora uses to create an Audio derivative: ).